recipe/build.sh

```diff
@@ -4,28 +4,35 @@ set -e
 echo "**************** M F R O N T  B U I L D  S T A R T S  H E R E ****************"
 
 # https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html#an-aside-on-cmake-and-sysroots
+if [[ "${target_platform}" == osx-* ]]; then
+  export LDFLAGS="$LDFLAGS -lm -lpthread -ldl -lz -lomp"
+  export LIBPATH="$PREFIX/lib $LIBPATH"
+  export SUFFIX=.dylib
+else
+  export LDFLAGS="$LDFLAGS -L$PREFIX/lib -lm -lpthread -lrt -ldl -lz -lgomp"
+  export LIBPATH="$PREFIX/lib $LIBPATH"
+  export SUFFIX=.so
+fi
 
-#export LDFLAGS="--sysroot ${CONDA_BUILD_SYSROOT} -L$PREFIX/lib -L${CONDA_BUILD_SYSROOT}/lib64 -lm -lpthread -L${CONDA_BUILD_SYSROOT}/usr/lib64 -lrt -ldl -L${PREFIX}/lib -lz -lgomp"
-export LDFLAGS="-L$PREFIX/lib -lm -lpthread -lrt -ldl -lz -lgomp"
-#export CXXFLAGS="${CXXFLAGS} -I${PREFIX}/include -w"
-export LIBPATH="$PREFIX/lib $LIBPATH"
-cmake    -Wno-dev \
+cmake ${CMAKE_ARGS} -Wno-dev \
```

Comment:
```suggestion
export LIBPATH="$PREFIX/lib $LIBPATH"
cmake ${CMAKE_ARGS} -Wno-dev \
```

This was common to both the if and else, easiest to keep it common.